### PR TITLE
Import imapc_features documentation  and mention it in shared mailbox page

### PR DESCRIPTION
**Pull request by vmo69**
DOV-4443

### CODE REVIEW

**Comment by slusarz** [comment]
The core.rst changes are duplicative of MR #219 - specifically, https://github.com/dovecot/documentation/pull/219/commits/0eb58a391e800735d7c152afa352df76c20f0616

**Comment by vmo69** [comment]
> The core.rst changes are duplicative of MR #219 - specifically, [0eb58a3](https://github.com/dovecot/documentation/commit/0eb58a391e800735d7c152afa352df76c20f0616)

Ok i didn't see that earlier, i removed the first commit.

File: source/configuration_manual/shared_mailboxes/cluster_setup.rst [hunk 1]
```diff
@@ -62,6 +62,9 @@ Additionally imapc must be configured accordingly on the backends:
    configured on all backends and directors
  * :ref:`setting-imapc_host` must point to a load balancer's address that
    connects to Dovecot director/proxy
+ * :ref:`setting-imapc_features`: To be able to pass through ``SEARCH`` commands
+   to the sharing users backend :ref:`setting-imapc_features` should at least
```
Comment (sirainen) [inline source/configuration_manual/shared_mailboxes/cluster_setup.rst:66]: `users'` I think?

**Review by sirainen — DISMISSED** [review]

File: source/configuration_manual/shared_mailboxes/cluster_setup.rst (hunk 1 above)
Comment (cmouse) [reply source/configuration_manual/shared_mailboxes/cluster_setup.rst]: it now has `user's` but timo suggested `users'`

Comment (sirainen) [reply source/configuration_manual/shared_mailboxes/cluster_setup.rst]: user's looks to be more correct actually. But a comma would clarify in `user's backend, imapc_features ...`

**Review by cmouse — APPROVED** [review]

**Pull request merged**